app/controllers/support_interface/course_options_controller.rb

```diff
@@ -0,0 +1,7 @@
+module SupportInterface
+  class CourseOptionsController < SupportInterfaceController
+    def index
+      @course_options = CourseOption.where('vacancy_status != ?', 'vacancies').includes(:course, :site)
```

Comment:
This includes everything non-vacancies to also surface the course options that have a different status accidentally.